docs/lib/commandlist.ts

```diff
@@ -0,0 +1,61 @@
+export const links = [
+  {
+    name: 'Links',
+    items: [
+      {
+        name: 'Slack',
+        href: 'https://reservix.slack.com/archives/C02727BNZ3J',
+        keywords: ['help'],
+      },
+      {
+        name: 'Jira',
+        href: 'https://reservix.atlassian.net/jira/software/projects/DST/boards/134',
+      },
+      {
+        name: 'Figma Core Kit',
+        href: 'https://www.figma.com/design/NbTUW9zk15nN8idlfsEttS/%F0%9F%8C%BC-Marigold-CORE?t=VfTLYo5foFEjRxFY-0',
+        keywords: ['design'],
+      },
+      {
+        name: 'Support Center',
+        href: 'https://reservix.atlassian.net/servicedesk/customer/portal/77',
+        keywords: ['help'],
+      },
+      {
+        name: 'Storybook',
+        href: 'https://marigold-latest.vercel.app/?path=/story/components-accordion--basic',
+      },
+      {
+        name: 'Stackblitz',
```

Comment:
Rename to "Marigold Playground"?

Reply:
but do you really think when the people talk about it they mean Marigold Playground? I also added keyword for playground.